vagrant/ansible/roles/client.prep/tasks/centos8.yml

```diff
@@ -1,14 +1,3 @@
-- name: Install required packages
-  yum:
-    name:
-      - glusterfs-fuse
-      - cifs-utils
-      - samba-test
-      - python3-pip
-      - git
-      - make
```

Comment:
These seem pretty generic and I note that pip is used in the block immediately following this one. Maybe keep these few here?

Reply:
You are right. Except for `glusterfs-fuse` it will be better to keep the other ones here. I'll update it.